docs/supported-libraries.md

```diff
@@ -186,26 +186,26 @@ These are the supported libraries and frameworks:
 
 These are the application servers that the smoke tests are run against:
 
-| Application server                                                                    | Version                                  | JVM                                            | OS                                    |
-|---------------------------------------------------------------------------------------|------------------------------------------|------------------------------------------------|---------------------------------------|
-| [Jetty](https://www.eclipse.org/jetty/)                                               | 9.4.53                                   | OpenJDK 8, 11, 17, 21<br/>OpenJ9 8, 11, 17, 21 | [`ubuntu-latest`], [`windows-latest`] |
-| [Jetty](https://www.eclipse.org/jetty/)                                               | 10.0.19, 11.0.19                         | OpenJDK 11, 17, 21<br/>OpenJ9 11, 17, 21       | [`ubuntu-latest`], [`windows-latest`] |
-| [Jetty](https://www.eclipse.org/jetty/)                                               | 12.0.6                                   | OpenJDK 17, 21<br/>OpenJ9 17, 21               | [`ubuntu-latest`], [`windows-latest`] |
+| Application server                                                                    | Version                                  | JVM                                                    | OS                                    |
```

Comment:
Seems that this modification breaks the table formatting. According to previous @laurit suggestion.
When editing ensure that the | symbols that separate columns align and use intellij to reformat the table.

Reply:
fixed